### PR TITLE
Landing

### PR DESCRIPTION
**Pull request by cheedinmar**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,8 +8,14 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
+    "autoprefixer": "^9.8.8",
     "core-js": "^3.6.5",
-    "vue": "^3.0.0"
+    "postcss": "^7.0.39",
```
Comment (teebarg) [inline package.json:13]: remove, already in dev dependencies

File: package.json [hunk 2]
```diff
@@ -8,8 +8,14 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
+    "autoprefixer": "^9.8.8",
```
Comment (teebarg) [inline package.json:11]: remove, already in dev dependencies

File: package.json [hunk 3]
```diff
@@ -8,8 +8,14 @@
...
     "core-js": "^3.6.5",
-    "vue": "^3.0.0"
+    "postcss": "^7.0.39",
+    "tailwindcss": "npm:@tailwindcss/postcss7-compat@^2.2.17",
```
Comment (teebarg) [inline package.json:14]: remove, already in dev dependencies

File: package.json [hunk 4]
```diff
@@ -8,8 +8,14 @@
...
+    "vue": "^3.0.0",
+    "vue-cli-plugin-tailwind": "^2.2.18",
```
Comment (teebarg) [inline package.json:16]: remove, already in dev dependencies

File: src/App.vue [hunk 5]
```diff
@@ -1,20 +1,18 @@
...
 </template>
 
 <script>
-import HelloWorld from './components/HelloWorld.vue'
-
-export default {
-  name: 'App',
-  components: {
-    HelloWorld
+import nav_bar from "./components/nav_bar.vue"
```
Comment (teebarg) [inline src/App.vue:10]: let's use convection `NavBar`

File: src/App.vue [hunk 6]
```diff
@@ -1,20 +1,18 @@
...
+import foot_er from "./components/footer.vue"
```
Comment (teebarg) [inline src/App.vue:11]: Same here

File: src/assets/svgs/banner.vue [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
...8/fYAAGhKCgAAJu3aa69KHYEc6OzsjI9//MOWAhTIQL8JAIBmpAAAYNJ6LAPgJKcCFIsJAIDmpAAAYNJ+4d1v9Z4vP2UpQHEcPXosdQQAElAAAHBOVq16TeoI5ISlAACQbwoAAM7Jtm3Xp45AjlgKUBwPPvBIPXUGAGaWAgCAc7K1RwHAS1kKUAx99gEAaDoKAADOyebN12SzZnWnjkGOWApQDP39g6kjADDDFAAAnLPrrtuYOgI5YylA/g0oAACajgIAgHPWYxkAp2EpQL71DwyljgDADFMAAHDOfv4ddxr25hU6OzvjYx/9oKUAOdVvDwCApqMAAGBKrF17SeoI5NC1m66yFCCnBvpNAAA0GwUAAFNix44bUkcgpywFyCcTAADNpy11AGg2P/rRU/XeZ55NHQOm3IEDz0epvTN1DKbYN79xd8yZM+ucv84dP/f6+Oxn/ioirAfIi5EXx+JvvnxXvb29PXUUmtQdd97iGwLMMP/oYIb96Z/8Rf1Lf/2V1DEAAJL6/F/+UVy0crn7EZhBJgBghq1dd0m84fYdqWPAtBgZGYlqtZY6BlOou7t7yjbxq1ZrMTIykvopcYqurq5oaXH/RRpu/gEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABrT/wdzwCi7ZAWuMwAAACV0RVh0ZGF0ZTpjcmVhdGUAMjAyMS0xMi0wNVQyMTowOTo1MiswMDowMG+syzwAAAAldEVYdGRhdGU6bW9kaWZ5ADIwMjEtMTItMDVUMjE6MDk6NTIrMDA6MDAe8XOAAAAAAElFTkSuQmCC"/>
```
Comment (teebarg) [inline src/assets/svgs/banner.vue:8]: what kind of svg is this?

File: src/main.js [hunk 8]
```diff
@@ -1,5 +1,12 @@
-import { createApp } from 'vue'
-import App from './App.vue'
-import './assets/css/style.css'
+import { createApp } from "vue";
+import App from "./App.vue";
+import "./index.css";
```
Comment (teebarg) [inline src/main.js:3]: remove

File: package.json (hunk 1 above)
Comment (cheedinmar) [reply package.json:13]: done

File: src/assets/svgs/banner.vue (hunk 7 above)
Comment (cheedinmar) [reply src/assets/svgs/banner.vue:8]: image

File: src/components/NavBar.vue [hunk 9]
```diff
@@ -0,0 +1,134 @@
...
+export default {
+  data(){
+    return{
+      state:false
+    }
+  },
+  methods:{
+     toggleHamburger() {
+      this.state = !this.state;
+      console.log(this.$refs.hamburger);
```
Comment (teebarg) [inline src/components/NavBar.vue:120]: remove log

File: src/components/NavBar.vue [hunk 10]
```diff
@@ -0,0 +1,134 @@
...
+    },
+    // clickaway(e){
```
Comment (teebarg) [inline src/components/NavBar.vue:122]: remove unused code

File: package.json [hunk 11]
```diff
@@ -9,7 +9,10 @@
   },
   "dependencies": {
     "core-js": "^3.6.5",
-    "vue": "^3.0.0"
+    "vue": "^3.0.0",
+    "vue-cli-plugin-tailwind": "^2.2.18",
```
Comment (teebarg) [inline package.json:13]: remove this

**Pull request merged**